protobuf/model_config.proto

```diff
@@ -219,6 +256,12 @@ message ModelInstanceGroup
   //@@
   repeated int32 gpus = 3;
 
+  //@@  .. cpp:var:: SecondaryDevice secondary_devices (repeated)
+  //@@
+  //@@     Secondary devices to run this instance group on. Optional.
```

Comment:
"Secondary devices that are required by instances specified by this instance group. Optional."